shared/tokenamount/tokenamount.go

```diff
@@ -60,54 +69,63 @@ func BigFromString(s string) (TokenAmount, error) {
 	return TokenAmount{v}, nil
 }
 
-func BigMul(a, b TokenAmount) TokenAmount {
+// Mul multiples two token amounts
+func Mul(a, b TokenAmount) TokenAmount {
 	return TokenAmount{big.NewInt(0).Mul(a.Int, b.Int)}
 }
 
-func BigDiv(a, b TokenAmount) TokenAmount {
+// Div divides two token amounts
+func Div(a, b TokenAmount) TokenAmount {
 	return TokenAmount{big.NewInt(0).Div(a.Int, b.Int)}
 }
 
-func BigMod(a, b TokenAmount) TokenAmount {
+// Mod computes the remainder of two token amounts
+func Mod(a, b TokenAmount) TokenAmount {
 	return TokenAmount{big.NewInt(0).Mod(a.Int, b.Int)}
 }
 
-func BigAdd(a, b TokenAmount) TokenAmount {
+// Add adds two token amounts together
+func Add(a, b TokenAmount) TokenAmount {
 	return TokenAmount{big.NewInt(0).Add(a.Int, b.Int)}
 }
 
-func BigSub(a, b TokenAmount) TokenAmount {
+// Sub subtracts the second token amount from the first
+func Sub(a, b TokenAmount) TokenAmount {
 	return TokenAmount{big.NewInt(0).Sub(a.Int, b.Int)}
 }
 
-func BigCmp(a, b TokenAmount) int {
+// Cmp compares two token amounts (for sorting)
+func Cmp(a, b TokenAmount) int {
 	return a.Int.Cmp(b.Int)
 }
 
-func (bi TokenAmount) Nil() bool {
-	return bi.Int == nil
+// Nil is true if there is no underlying token amount
+func (ta TokenAmount) Nil() bool {
+	return ta.Int == nil
 }
 
 // LessThan returns true if bi < o
-func (bi TokenAmount) LessThan(o TokenAmount) bool {
-	return BigCmp(bi, o) < 0
+func (ta TokenAmount) LessThan(o TokenAmount) bool {
```

Comment:
This comment should say, `ta < o`, and same for the following couple of methods.